src/components/Shared/CrossingMapPage/CrossingMapSearchBar.js

```diff
@@ -91,9 +91,17 @@ class CrossingMapSearchBar extends Component {
     { suggestion, suggestionValue, suggestionIndex, sectionIndex, method },
   ) => {
     if (suggestion.__typename === 'Crossing') {
-      this.props.history.push(`/map/crossing/${suggestion.id}`);
+      if(this.props.location.pathname.includes('dashboard')) {
```

Comment:
might be nice to do this logic in one place and set a var here, looks like we're checking in two places